.readthedocs.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+formats: all
+sphinx:
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+build:
+  os: ubuntu-22.10
+  tools:
+    python: "3.7"  # use min version for async-lru==1.0.3
```

Comment:
Why not use recent Python to build the docs?

Reply:
Updated in https://github.com/scrapinghub/web-poet/pull/147/commits/7ddf27624f43aab6a005a921878525ede1781a77